sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs

```diff
@@ -431,7 +425,6 @@ Page<MetricSeriesData> FirstPageFunc(int? pageSizeHint)
         /// <returns></returns>
         public virtual AsyncPageable<EnrichmentStatus> GetMetricEnrichmentStatusesAsync(string metricId, GetMetricEnrichmentStatusesOptions options, CancellationToken cancellationToken = default)
```

Comment:
asking in case you missed it. Are you going to update this and the equivalent method docstrings in a future PR?

Reply:
Yes. We have an issue tracking this (https://github.com/Azure/azure-sdk-for-net/issues/16047).